CMakeLists.txt

```diff
@@ -9,9 +9,17 @@ project(
     LANGUAGES CXX
 )
 
-include(CTest)
+if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
+    message(FATAL_ERROR "In-source builds are not allowed!")
+endif()
+
+include(CPack)
 include(FetchContent)
+include(GNUInstallDirs)
 
+add_subdirectory(src/beman/exemplar)
+
+option(BUILD_TESTING "build tests too" ${PROJECT_IS_TOP_LEVEL})
```

Comment:
The recommended pattern for options is to include the name of the project, so that projects can compose. Defaulting to is top level is the right thing, but we wouldn't want to build the tests of beman_interface if we included that. 
Was going to come back to fix, but if we're touching this, might as well.

Reply:
@ClausKlein , please update this PR with latest main.

Reply:
done